cli/running/running.go

```diff
@@ -155,6 +155,19 @@ loop:
 	return result
 }
 
+// createDataDir checks if DataDir folder exists, if not creates it.
+func createDataDir(dataDirPath string) error {
+	_, err := os.Stat(dataDirPath)
+	if os.IsNotExist(err) {
+		err = os.Mkdir(dataDirPath, 644)
```

Comment:
Write in the comments what 644 means. Seriously, it's not easy to remember what rights it grants.

Reply:
Added description